modules/microshift-checking-pods-mismatch.adoc

```diff
@@ -0,0 +1,161 @@
+
+[IMPORTANT]
+====
+* Replace `_<pod_name_A>_` with the name of the first pod in the following procedure.
+* Replace `_<pod_name_B>_` with the name of the second pod in the following procedure.
+* Replace `_<PVC_mountpoint>_` with the mount point within the pods.
+====
+
+.Procedure
+
+. List the mount point within the first pod by running the following command:
++
+[source,terminal]
+[subs="+quotes"]
+----
+$ oc get pods -n _<pod_name_A>_ -ojsonpath='{.spec.containers[*].volumeMounts[*].mountPath}' <1>
+----
+<1> Replace `_<pod_name_A>_` with the name of the first pod.
++
+.Example output
+[source,terminal]
+----
+/files /var/run/secrets/kubernetes.io/serviceaccount
+----
+. List the mount point within the second pod by running the following command:
++
+[source,terminal]
+[subs="+quotes"]
+----
+$ oc get pods -n _<pod_name_B>_ -ojsonpath='{.spec.containers[*].volumeMounts[*].mountPath}' <1>
+----
+<1> Replace `_<pod_name_B>_` with the name of the second pod.
++
+.Example output
+[source,terminal]
+----
+/files /var/run/secrets/kubernetes.io/serviceaccount
+----
+. Check the context and permissions inside the first pod by running the following command:
++
+[source,terminal]
+[subs="+quotes"]
+----
+$ oc rsh _<pod_name_A>_ ls -lZah _<PVC_mountpoint>_ <1>
+----
+<1> Replace `_<pod_name_A>_` with the name of the first pod and replace `_<PVC_mountpoint>_` with the mount point within the first pod.
++
+.Example output
+[source,terminal]
+----
+total 12K
+dr-xr-xr-x.   1 root root system_u:object_r:container_file_t:s0:c398,c806   40 Feb 17 13:36 .
+dr-xr-xr-x.   1 root root system_u:object_r:container_file_t:s0:c398,c806   40 Feb 17 13:36 ..
+[...]
+----
+. Check the context and permissions inside the second pod by running the following command:
++
+[source,terminal]
+[subs="+quotes"]
+----
+$ oc rsh _<pod_name_B>_ ls -lZah _<PVC_mountpoint>_ <1>
+----
+<1> Replace `_<pod_name_B>_` with the name of the second pod and replace `_<PVC_mountpoint>_` with the mount point within the second pod.
++
+.Example output
+[source,terminal]
+----
+total 12K
+dr-xr-xr-x.   1 root root system_u:object_r:container_file_t:s0:c15,c25   40 Feb 17 13:34 .
+dr-xr-xr-x.   1 root root system_u:object_r:container_file_t:s0:c15,c25   40 Feb 17 13:34 ..
+[...]
+----
+. Compare both the outputs to check if there is a mismatch of seLinux context.
+. In case of mismatch of the seLinux content, create a new SCC and assign it to both PODs. To create SCC see the link: https://docs.openshift.com/container-platform/4.15/authentication/managing-security-context-constraints.html#security-context-constraints-creating_configuring-internal-oauth[].
+. Update the seLinuxContext as shown in the following example:
++
+.Example output
+[source,terminal]
+----
+ [...]
+ securityContext:privileged
+      seLinuxOptions:MustRunAs
+        level: "s0:cXX,cYY"
+  [...]
+----
+
+.Verification
```

Comment:
Verifications are really supposed to be 1 or 2 steps. If it makes sense, you can place these into the procedure after each related task, or perhaps place in to a separate module.

Reply:
Modularization is taken up in the jira [OSDOCS-12154](https://issues.redhat.com/browse/OSDOCS-12154). All other changes are included in this PR.